examples/magento-graphcms/pages/[...url].tsx

```diff
@@ -73,13 +74,12 @@ function CategoryPage(props: CategoryProps) {
         canonical={page?.url ? `/${page.url}` : undefined}
         {...category}
       />
-
       <LayoutHeader floatingMd>
         <LayoutTitle size='small' component='span'>
           {category?.name ?? page.title}
         </LayoutTitle>
       </LayoutHeader>
-
+      <ProductPageCategoryBreadcrumb {...categories?.items?.[0]} />
```

Comment:
This should be placed somewhere on the page where it isn't occluded by the `< Back` button, we can probably discuss where this should go, even if we don't included it here in the final PR.